pkgs/by-name/ol/ollama/package.nix

```diff
@@ -199,6 +199,13 @@ goBuild {
       make ${dist_cmd} -j $NIX_BUILD_CORES
     '';
 
+  postInstall = lib.optionalString stdenv.hostPlatform.isLinux ''
```

Comment:
@GaetanLepage  just wondering, why was this removed in the first place? Want to make sure I'm not overlooking anything by adding it back